include/pybind11/numpy.h

```diff
@@ -14,12 +14,14 @@
 
 #include <algorithm>
 #include <array>
+#include <cassert>
 #include <cstdint>
 #include <cstdlib>
 #include <cstring>
 #include <functional>
 #include <numeric>
 #include <sstream>
+#include <stdalign.h>
```

Comment:
Remove this, this is a C interop header (and we're not writing C).

Reply:
Done: 109a1659c881d4c569430add6d3e170b44599eb8